### PR TITLE
[improvements]-testionials_cards

### PR DESCRIPTION
**Pull request by CaneteJoash**
## Related Links
https://www.notion.so/Project-UI-UX-Capstone-716c808e203c4cce85691202f6bd1eca?p=ef8e692ffe604c5495fd35b6ee65d509

## Description
- Reduce testimonials cards in large screens

## Screenshots
image
image
image

### CODE REVIEW

**Review by ume45876139 — APPROVED** [review]
# Working

<details>
<summary>PC</summary>


https://user-images.githubusercontent.com/39412730/166187166-4882dc0f-a3a9-44c3-80c1-61734694d2a4.mp4


</details>

<details>
<summary>Tablet</summary>

https://user-images.githubusercontent.com/39412730/166187199-7c6e6efd-d885-474b-bcc7-628d9840967e.mp4


</details>


<details>
<summary>Mabile</summary>


https://user-images.githubusercontent.com/39412730/166187227-575b43f6-204e-4990-ad28-e3e3f4df3023.MP4


</details>

- Testimonial card size is proper

# Testing data
test device: Chrome, Safari
test time: 0.25h
comment: 0
iteration: 0
total test time: 0.25h

File: src/utilities/constants.js [hunk 1]
```diff
@@ -48,6 +48,24 @@ export const testimonials = [
     text: "Never thought I'd get interested in an app like this. I love Swipet.",
     author: 'Joash',
   },
+  {
+    name: 'Woman with cat.',
+    src: '/images/Test03.jpg',
+    text: 'I really fund my happiness here.',
```
Comment (charlotteds) [inline src/utilities/constants.js]: Checking spelling for `find`.

**Review by charlotteds — COMMENTED** [review]
- Rename the images so they follow our naming convention, starting with an uppercase letter (Example: Test03.jpg)
- Please resize and compress the images, without breaking the quality. 
Web apps perform better with images that are 200KB or smaller in file size.
You can check the images for the other testimonials. 
Dimension: 800x800
File size: <= 200KB
_Note: The hero image is a special case. :)_

You can use this to resize/compress images: https://tinypng.com/
Check our images here: https://drive.google.com/drive/folders/1wKQkSUpiw2PdecrPx5NIrvLB1_Pp4YYY

File: src/pages/components/testimonial/index.js [hunk 2]
```diff
@@ -46,18 +58,20 @@ const Testimonial = () => {
...
+          >
+            <PreviousIcon />
+          </button>
+          <button
+            className='group slider-control'
+            onClick={() => sliderRef.slickNext()}
+          >
+            <NextIcon />
+          </button>
+        </div>
```
Comment (charlotteds) [inline src/pages/components/testimonial/index.js]: Remove unnecessary `div`.

**Comment by roseaugusto** [comment]
@CaneteJoash 
1. Please remove the `px-5` instead use `pr-5` something to prevent extra padding in the left.
image

2. Sorry for this but I realize, it's a bit small now so please revert back to this.
image

File: src/assets/css/testimonials.css [hunk 3]
```diff
@@ -1,3 +1,8 @@
 .slider-control {
   @apply rounded-full w-[30px] h-[30px] my-3 ml-5 sm:ml-2 bg-navy-300 hover:bg-navy-500 focus:bg-navy-100;
 }
+
+
+.slick-track{
+  margin-left: 0px !important;
+}
```
Comment (charlotteds) [inline src/assets/css/testimonials.css]: - Remove extra space between classes.
- You can use tailwind class for the margin:
```suggestion
.slick-track{
  @apply ml-0 !important;
}
```

**Review by charlotteds — COMMENTED** [review]
- Rename the images so they follow our naming convention, starting with an uppercase letter (Example: Test03.jpg)

**Review by roseaugusto — APPROVED** [review]
Approve!

**Review by charlotteds — APPROVED** [review]

**Pull request merged**